acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/keymanager/DeleteKeystoresAcceptanceTest.java

```diff
@@ -67,10 +67,23 @@ public void invalidRequestBodyReturnsError() throws URISyntaxException {
   }
 
```

Comment:
What about an AT showing that 2nd delete returns not_active if there is slashing protection data?

Reply:
That test already exists: https://github.com/ConsenSys/web3signer/blob/dba2a0614cb81080d0180a4b221348500ebd0a23/acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/keymanager/DeleteKeystoresAcceptanceTest.java#L97-L120